<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchResultsDialog</class>
 <widget class="QDialog" name="SearchResultsDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1004</width>
    <height>638</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Search results</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Search results</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTableWidget" name="resTable">
        <property name="columnCount">
         <number>6</number>
        </property>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>120</number>
        </attribute>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>35</number>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>30</number>
        </attribute>
        <column>
         <property name="text">
          <string>Text</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Block number</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Offset</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Encoding</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>File type</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>File path</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QProgressBar" name="m_progressBar">
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_pause">
       <property name="text">
        <string>Pause</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_cancel">
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_rescan">
       <property name="text">
        <string>Restart</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="m_erase">
       <property name="text">
        <string>Erase found items</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveReport">
       <property name="text">
        <string>Save report</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>cellclicked(int,int)</slot>
 </slots>
</ui>
